lib/v2/shisu/news.js

```diff
@@ -0,0 +1,43 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const { parseDate } = require('@/utils/parse-date');
+const timezone = require('@/utils/timezone');
+
+const url = 'https://news.shisu.edu.cn';
+
+module.exports = async (ctx) => {
+    const { data: res } = await got(String(url));
+    const $ = cheerio.load(res);
+    const itemsoup = $('#gallery-wrapper > article')
+        .toArray()
+        .map((i0) => {
+            const i = $(i0);
+            return {
+                title: i.find('a').text(),
+                link: `${url}${i.find('a').attr('href')}`,
+                pubDate: timezone(parseDate(i.find('.in-con02 > span:nth-child(2)').text(), 'YYYY-MM-DD'), +8),
+                description: i.find('.default-txt').text(),
+                language: 'zh-cn',
+                image: i.find('img').attr('src'),
+                icon: 'https://www.shisu.edu.cn/assets/images/sisu.ico',
```

Comment:
`item` in an RSS feed does not contain these fields.
```suggestion
```
docs: https://docs.rsshub.app/joinus/advanced/advanced-feed